starter-code/scripts/article.js

```diff
@@ -50,13 +51,39 @@ Article.fetchAll = function() {
     // When rawData is already in localStorage,
     // we can load it with the .loadAll function above,
     // and then render the index page (using the proper method on the articleView object).
-    Article.loadAll(); //TODO: What do we pass in to loadAll()?
+    Article.loadAll(JSON.parse(localStorage.getItem('rawData'))); //TODO: What do we pass in to loadAll()? DONE
     //TODO: What method do we call to render the index page?
+    // DONE, took about 1 minute
+    articleView.initIndexPage();
```

Comment:
IF you were to put this call in the loadAll function, would you want to call it here again?